Index: .idea/.gitignore
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
deleted file mode 100644
--- a/.idea/.gitignore	(revision 9d70d3af4c0b6a358caf9e560cd0f46dd98b829f)
+++ /dev/null	(revision 9d70d3af4c0b6a358caf9e560cd0f46dd98b829f)
@@ -1,5 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
-# Environment-dependent path to Maven home directory
-/mavenHomeManager.xml
